reset-password.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Reset Password</title>
+</head>
+<body>
+    <form action="">
+        <label>Reset Password</label>
+        <input type="password">
```

Comment:
Good job. Can we add a submit button?

Reply:
Added! Let me know if you need anything else.